src/main/java/edu/cornell/kfs/concur/service/impl/ConcurFailedEventQueueProcessingController.java

```diff
@@ -0,0 +1,39 @@
+package edu.cornell.kfs.concur.service.impl;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+import org.kuali.kfs.sys.KFSConstants;
+
+import edu.cornell.kfs.rass.RassConstants;
+
+public enum ConcurFailedEventQueueProcessingController {
```

Comment:
Naming this a "Controller" seems a little confusing, since it's neither a Kuali presentation controller nor a Spring MVC controller. Could this be renamed to something a little clearer (like a "Mode")?

Reply:
sure, I can change to mode